terraso_backend/apps/graphql/schema/schema.graphql

```diff
@@ -636,7 +636,7 @@ type ProjectNode implements Node {
   updatedAt: DateTime!
   name: String!
   description: String!
-  group: GroupNode!
+  membershipList: CollaborationMembershipListNode!
```

Comment:
nit: it would be nice if there were project specific types for `MembershipList` and `Membership` with the project model's role types, but i'm not sure how that'd work with graphene

Reply:
Good point, I'll give this a try.